src/Sylius/Bundle/ThemeBundle/Translation/Translator.php

```diff
@@ -47,14 +47,14 @@
     /**
      * @param TranslatorLoaderProviderInterface $loaderProvider
      * @param TranslatorResourceProviderInterface $resourceProvider
-     * @param MessageSelector $messageSelector
+     * @param MessageFormatterInterface $messageFormatter
      * @param string $locale
      * @param array $options
      */
     public function __construct(
         TranslatorLoaderProviderInterface $loaderProvider,
         TranslatorResourceProviderInterface $resourceProvider,
-        MessageSelector $messageSelector,
```

Comment:
🚨 BC break alert 🚨 

The way we can solve it here is to remove the typehint and do instanceof checks inside to accept `MessageSelector|MessageFormatterInterface`. We'll also need tests which use both of them.